notebooks/intro.jl

```diff
@@ -755,9 +1041,9 @@ Unitful = "~1.13.1"
 PLUTO_MANIFEST_TOML_CONTENTS = """
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.0-rc3"
```

Comment:
should we better use 1.8.5?

Reply:
I think it's better for us to use 1.9 in order to have fast startup

Reply:
Plus 1.9 was literally released this morning

Reply:
haha true!

then we just need to rerun it with 1.9.0 instead of rc3